python/packages/nisar/workflows/focus.py

```diff
@@ -1622,6 +1629,9 @@ def focus(runconfig, runconfig_path=""):
         dop[frequency] = scale_doppler(dop_ref, band.center / fc_ref)
         blocks_bounds[frequency] = plan_processing_blocks(cfg, ogrid[frequency],
                                         dop[frequency], dem, orbit)
+        # So does output pixel area (beta0 convention).
+        daz, _ = get_radar_grid_nominal_ground_spacing(ogrid[frequency], orbit)
```

Comment:
It looks like you are using a single `daz` value for each frequency. However, the variation in ground velocity across the radar grid may not be negligible. As you know, the ground velocity depends on the topography, but also on the off-nadir angle, which may be significant over a wide swath.

Reply:
@bhawkins , I created the issue #171 so we don't forget about this approximation.